mypyc/test-data/run-integers.test

```diff
@@ -328,3 +328,14 @@ def test_int_as_bool() -> None:
     for x in 1, 55, -1, -7, 1 << 50, 1 << 101, -(1 << 50), -(1 << 101):
         assert is_true(x)
         assert not is_false(x)
+
+[case testMin]
+def check_min_int() -> None:
+    x: int = 200
```

Comment:
I think functions with prefix `check` wouldn't trigger a run test? Maybe you should change it to `test_*`. Also, please add more run tests about other types, such as strings, floats.

Reply:
I'm having some issues with my system right now. I'll try to update the branch by the weekend.